example/setup.go

```diff
@@ -140,6 +140,4 @@ func Localstack(s *ocsns.SNS, q *ocsqs.SQS) error {
 	}
 
 	log.Println("Localstack: Created Subscription:", *sub.SubscriptionArn)
-
-	return nil
```

Comment:
I think actually this should return the `err` from `s.Subscribe` and kill the `if err != nil` check

Reply:
Yes, you're right and I have updated the code and do PR again..